lib/node_modules/@stdlib/stats/base/dists/gumbel/mgf/lib/main.js

```diff
@@ -33,7 +33,7 @@ var exp = require( '@stdlib/math/base/special/exp' );
 * @param {number} t - input value
 * @param {number} mu - mean
 * @param {PositiveNumber} beta - scale parameter
-* @returns {number} evaluated MGF
+* @returns {number} mgf
```

Comment:
Needs to be reverted:

```suggestion
* @returns {number} evaluated MGF
```